interpolator/interpolator.F90

```diff
@@ -445,14 +445,21 @@ subroutine interpolator_init( clim_type, file_name, lonb_mod, latb_mod, &
   read (input_nml_file, nml=interpolator_nml, iostat=io)
   ierr = check_nml_error(io,'interpolator_nml')
 
+  ! retain_cm3_bug is no longer supported.
+  if (retain_cm3_bug) then
+    if (mpp_pe() == mpp_root_pe()) call mpp_error(FATAL, "interpolator_init: You have overridden the default " // &
+       "value of retain_cm3_bug and set it to .true. in interpolator_nml. This was a temporary workaround " // &
+       "that is no longer supported. Please remove this namelist variable.")
+  endif
+
 !---------------------------------------------------------------------
 !    write version number and namelist to logfile.
 !---------------------------------------------------------------------
-call write_version_number("INTERPOLATOR_MOD", version)
+  call write_version_number("INTERPOLATOR_MOD", version)
 
-if (mpp_pe() == mpp_root_pe() ) write (stdlog(), nml=interpolator_nml)
+  if (mpp_pe() == mpp_root_pe() ) write (stdlog(), nml=interpolator_nml)
 
-module_is_initialized = .true.
+  module_is_initialized = .true.
```

Comment:
This is fine, BUT now you own these lines of code.  `git blame` will show up as you.